token/core/fabtoken/des.go

```diff
@@ -12,18 +12,23 @@ import (
 	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
 )
 
+// GetAuditorVerifier deserializes the verifier for the passed auditor identity
```

Comment:
why do we need this `des.go` file anyway? why is `deserializer.go` not enough?

Reply:
they implement different interfaces.

Reply:
I see they implement the same interface
What am I missing?

Reply:
deserializer implements in addition to the methods in Des, VerifierDes interface. This is used in the validator. Service instantiation uses the deserializer. There is potential for refactoring, but it is not going to  be addressed in this PR.